lib/db.js

```diff
@@ -14,7 +14,11 @@ module.exports = function () {
 
   // special mode, will emulate all redis commands.
   // designed for demo and test scenarious to avoid having real Redis instance
-  const emulate = process.argv[2] === 'emulate' || redisOptions.emulate;
+  let emulate = process.argv[2] === 'emulate' || redisOptions.emulate;
+
+  if (process.env.EG_DB_EMULATE) {
+    emulate = !!parseInt(process.env.EG_DB_EMULATE);
```

Comment:
I like `process.env.EG_DB_EMULATE`
can we make this resistant to string or  !!parseInt(true) !!parseInt("true") !!parseInt("emulate") is false

Reply:
Hopefully we will be able to remove this hack when configuration files will support env variables. When this will happen, that value will be injected directly from the CI environment.

Reply:
@DrMegavolt I would not spend time on this as this is not something we will let people use. I would neither document it.

It's just a temporary thing to make this PR happen. Once config files will support env variable, this piece is going to vanish.